github-pullrequest-plugin/src/test/resources/org/jenkinsci/plugins/github_integration/its/WorkflowITest/workflow-definition2.groovy

```diff
@@ -0,0 +1,31 @@
+package org.jenkinsci.plugins.github_integration.its.WorkflowITest
+
+
+try {
+    node('master') {
+        step([
+                $class       : 'GitHubPRStatusBuilder',
+                statusMessage: [
+                        content: "Run #${env.BUILD_NUMBER} started"
+                ]
+        ])
+        setGitHubPullRequestStatus state: 'PENDING', context: 'custom-context1', message: "Run #${env.BUILD_NUMBER} started"
+        setGitHubPullRequestStatus state: 'PENDING', context: 'custom-context2', message: "Run #${env.BUILD_NUMBER} started"
+
+        sh 'sleep 10 && env'
+
+        setGitHubPullRequestStatus state: 'SUCCESS', context: 'custom-context1', message: 'Tests passed'
+        setGitHubPullRequestStatus state: 'SUCCESS', context: 'custom-context2', message: 'Tests passed'
+        step([
```

Comment:
hook btw not needed because test usually happens behind the nat, create repo, push changes, open prs, etc permissions. Just try provide everything.